### DISCUSSION TITLE
"permission denied" on creating system VM with iso file in home directory

### DISCUSSION THREAD

**Discussion by 4aya** [cockpit-project/cockpit-machines#1110] [Q&A] [answered]
Hello, i have a small issue when i try to create vms
My os:
![image](https://github.com/cockpit-project/cockpit-machines/assets/115783539/ada18bc3-fb98-47c1-82fa-9ee686422286)

![image](https://github.com/cockpit-project/cockpit-machines/assets/115783539/81fa261c-f2d2-47a7-b308-e19a1f382464)
After that error, whole cockpit broke

![image](https://github.com/cockpit-project/cockpit-machines/assets/115783539/55ef6e35-33e6-41ee-9350-7ec47b9df85a)

**Comment by martinpitt** [comment, marked as answer]
You are trying to create this VM on the system libvirt? On the system there is probably an AppArmor policy forbidding this ("permission denied"). It may only allow files from /var/lib/libvirt/images/. Try to create it on the user libvirt and check if that works?

Then copy/move it to /var/lib/libvirt/images if you want/need it on the system level.
